Update the API that is suggsested for viewing a client's entire infra… (#8801)
DataDog/documentation: content/en/developers/guide/query-the-infrastructure-list-via-the-api.md

  - /developers/faq/query-the-infrastructure-list-via-the-api
---

> More advanced Datadog users may sometimes want to use [the API][1] to query general data about their infrastructure—the kind of data that you can find in your [infrastructure list][2] or the [host map][3]. You can do this via an API GET request on the [api/v1/hosts][4] endpoint.

Reviewer: ```suggestion
If you're a more advanced Datadog user, you may want to use [the API][1] to query general data about infrastructure—the kind of data that you can find in your [infrastructure list][2] or the [host map][3]. You can do this via an API GET request on the [api/v1/hosts][4] endpoint.
```

> If you're a more advanced Datadog user, you may want to use [the API][1] to query general data about infrastructure—the kind of data that you can find in your [infrastructure list][2] or the [host map][3]. You can do this via an API GET request on the [api/v1/hosts][4] endpoint.

Reviewer: ```suggestion
If you're a more advanced Datadog user, you may want to use [the API][1] to query general data about infrastructure—the kind of data that you can find in your [infrastructure list][2] or the [host map][3]. You can do this with an API GET request on the [api/v1/hosts][4] endpoint.
```
